package.json

```diff
@@ -207,6 +210,34 @@
           },
           "description": "Indexes all terraform sources to get live syntax errors, rename support, go to symbol, and much more..."
         },
+        "terraform.languageServer": {
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Enables or disables the indexing",
+              "default": false
+            },
```

Comment:
It seems like it might be more clear for users if we expose these config options as standard controls in extension settings versus having them manually edit `settings.json` and figure out what the available options are.

Is there a particular reason we have them all hidden in an `object` type? Could we maybe [expose the obvious ones](https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema), like `enabled`, via normal namespacing (i.e., `terraform.languageServer.enabled`)? This would apply to `terraform.indexing...` as well.

Reply:
I'm sticking with the existing settings infrastructure for now as I think there is already enough churn in this PR without refactoring the settings stuff. My recommendation is that this could be picked up in a different PR if @mauve agrees once back from hols.